src/core/electric_utility.jl

```diff
@@ -156,6 +156,11 @@ struct ElectricUtility
         outage_probabilities::Array{<:Real,1} = isempty(outage_durations) ? Float64[] : [1/length(outage_durations) for p_i in 1:length(outage_durations)],
         outage_time_steps::Union{Nothing, UnitRange} = isempty(outage_durations) ? nothing : 1:maximum(outage_durations),
         scenarios::Union{Nothing, UnitRange} = isempty(outage_durations) ? nothing : 1:length(outage_durations),
+
+        ### Grid Renewable Energy Fraction Inputs ###
+        # Utilities renewable energy fraction. Can be scalar or timeseries (aligned with time_steps_per_hour)
+        clean_energy_fraction_series::Union{Real, Array{<:Real, 1}} = Float64[],
+        cambium_cef_col::String = "cef_load",  # Column name for clean energy fraction in Cambium database
```

Comment:
can add options in the help text: Options: ["cef_load", "cef_gen"]

Reply:
Help text updated with Options: ["cef_load", "cef_gen"]